tests/fixtures.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from copy import deepcopy
-
 import pytest
-
+from sys import version_info
 from pydash._compat import iteritems
 
+if version_info[0] > 2:
```

Comment:
Would this approach work: http://stackoverflow.com/a/35230792

where you define the annotations in a comment?